sql/server.go

```diff
@@ -106,14 +106,28 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Pick up current session state.
+	// The request user is validated in ServeHTTP. Even in insecure mode,
+	// it is guaranteed not to be empty.
+	planMaker := planner{user: args.GetUser()}
+	// TODO(tschottdorf) will have to validate the Session information (for
```

Comment:
i think it's time to kill this comment. all our methods do the necessary validation down in the sql code